library/Zend/Config/Yaml.php

```diff
@@ -363,10 +365,10 @@ protected static function _parseValue($value)
 
         // remove quotes from string.
         if ('"' == $value['0']) {
-            if ('"' == $value[count($value) -1]) {
+            if ('"' == $value[strlen($value) -1]) {
```

Comment:
This is probably a long-standing bug, since `count` in this case would always return `1` on a string, so this would always be `$value[0]` which is the same as the condition above.